.docker/rootfs/entrypoint.sh

```diff
@@ -55,22 +54,10 @@ function handleStartup() {
     pwconv
   fi
 
-  if [ -e /use_apache ]; then
-    export APACHE_RUN_USER=$(id -nu "$USER_ID")
-    # This doesn't _exactly_ run as the specified GID, it runs as the GID of the specified user but WTF
-    export APACHE_RUN_GROUP=$(id -ng "$USER_ID")
-    export APACHE_PID_FILE=/var/run/apache2/apache2.pid
-    export APACHE_RUN_DIR=/var/run/apache2
-    export APACHE_LOCK_DIR=/var/lock/apache2
-    export APACHE_LOG_DIR=/var/log/apache2
-    export LANG=C
-  elif [ -e /use_fpm ]; then
-    sed -i "s/user = .*/user = $USER_ID/g" /usr/local/etc/php-fpm.d/www.conf
-    sed -i "s/group = .*/group = $GROUP_ID/g" /usr/local/etc/php-fpm.d/www.conf
```

Comment:
Are you sure we don't need these two configs any longer?

Reply:
I'm wondering why do we need to implement dynamic user and group. Woud't it be easier to create a user directry in the dockerfile and run the processes as this user?

Reply:
I have no clue. This was added by Toby (the previous docker maintainer).  I assume that we had permission problems. 
I can tell you what we need: 
- Kimai writes cache files during runtime and during image boot / installation, and that are different user (`root` vs `www-data`).
- Then there are the mounted directories (especially `var/data/`) which needs to be writable during runtime by `www-data`

Reply:
ok. I will take a look. I'm doing final touches to the PR. Sorry for the delay.

Reply:
> ok. I will take a look. I'm doing final touches to the PR. Sorry for the delay.

Please don't feel obligated at all. I am really really thankful for your input! 👍 
And btw: we can still write via email, which you find here: https://www.kevinpapst.de/ueber-mich/